pkgs/by-name/ja/jawiki-all-titles-in-ns0/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, fetchurl
+, stdenvNoCC
+}:
+
+let
+date = "20240520";
+jawiki = fetchurl {
+  url ="https://dumps.wikimedia.org/jawiki/${date}/jawiki-${date}-all-titles-in-ns0.gz";
+  hash= "sha256-HMQutYuKiI7fXmbivS0mIgdHsQu6t6WdVFVrnyw8JIw=";
+};
+in stdenvNoCC.mkDerivation {
+  pname = "jawiki-all-titles-in-ns0";
+  version = date;
+
+  unpackPhase = "true";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir $out
+    cp ${jawiki} $out/jawiki-all-titles-in-ns0.gz
```

Comment:
Why not just make `jawiki` the `src` of the derivation?

Reply:
Now that I think about it, I made every dictionary a derivation so it can be updated easily with auto-update scripts, but there is no easy way to update this automatically, so I think will add it to merge-ut-dictionaries.